Scenario5/Scenario5/Program.cs

```diff
@@ -0,0 +1,88 @@
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Scenario5
+{
+    class Program
+    {
+        // Do not modify this method.
+        static void Main(string[] args)
+        {
+            // Task 1
```

Comment:
Similarly, remove these comments. The TODO is what users should focus on and adding "Tasks" might confuse them.